.ci/pitest.sh

```diff
@@ -37,7 +37,9 @@ pitest-annotation|pitest-design \
 |pitest-packagenamesloader \
 |pitest-common-2|pitest-misc|pitest-xpath)
   mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
-  declare -a ignoredItems=();
+  declare -a ignoredItems=(
+    "OrderedPropertiesCheck.java.html:<td class='covered'><pre><span  class='survived'>            if (!prevKeyFound &#38;&#38; prevKeyPattern.matcher(line).matches()) {</span></pre></td></tr>"
```

Comment:
You can't place this in this area as this is for pitest that produce no results.
You must move your specific pitest out like the others below this.